src/game/clients/CClientEvent.cpp

```diff
@@ -1539,7 +1539,7 @@ void CClient::Event_VendorSell(CChar* pVendor, const VendorItem* items, uint uiI
 		}
 		else
 		{
-			if ( pVendor->IsStatFlag(STATF_PET) && pContExtra )
```

Comment:
Instead of changing this long-present default behavior, could you add a ini flag to enable it?

Reply:
> Instead of changing this long-present default behavior, could you add a ini flag to enable it?

Never did but I could try. Anyway I would like to know why player's vendors are allow to use the pContExtra. It is weird


> Is that happening only for player vendors or also for npcs?

Players vendors are allowed but I think they shouldnt.
 I think someone introduce this "long feature" while trying to deny player's vendors to use pContExtra. But he did the opposite and forget. OR does players vendors need pcontextra?